docs/connectors/getting-started-connectors/README.adoc

```diff
@@ -70,115 +72,167 @@ ifdef::qs[]
 Learn how to create and set up connectors in {product-long-connectors}.
 
 [#introduction]
-Welcome to the quick start for {product-long-connectors}. In this quick start, you'll learn how to create a source connector and sink connector and send data to and from {product-kafka}. A source connector allows you to send data from an external system to {product-kafka}. A sink connector allows you to send data from {product-kafka} to an external system. 
+Welcome to the quick start for {product-long-connectors}. 
+
+In this quick start, you learn how to create a source connector and sink connector and send data to and from {product-kafka}. 
+
+A *source* connector allows you to send data from an external system to {product-kafka}. A *sink* connector allows you to send data from {product-kafka} to an external system. 
+
+endif::[]
+
+ifndef::qs[]
+== Overview
+
+{product-long-kafka} is a cloud service that simplifies the process of running Apache Kafka. Apache Kafka is an open-source, distributed, publish-subscribe messaging system for creating fault-tolerant, real-time data feeds.  
+
+You can use {product-connectors} to configure communication between {product-kafka} instances and external services and applications. {product-connectors} allow you to configure how data moves from one endpoint to another without writing code. 
+
```

Comment:
We don't display images in the in-product quick starts, so the text explaining the diagram should be conditionalized to not be displayed in the quick start.

Also, in the line above the image, add the `[.screencapture]` metadata.

Reply:
That entire overview section is conditionalized to not display in the quick start - including the image

do I use [.screencapture] even if the image is not a screen capture but rather a diagram?